app/web/features/profile/view/NewHostRequest.tsx

```diff
@@ -208,18 +201,18 @@ export default function NewHostRequest({
           <TextField
             id="text"
             className={classes.requestField}
-            label={REQUEST}
+            label={t("global:request")}
             name="text"
             minRows={6}
             inputRef={register}
             multiline
             fullWidth
-            placeholder={REQUEST_DESCRIPTION}
+            placeholder={t("global:request_description")}
           />
           <CardActions className={classes.send}>
-            <Button onClick={() => setIsRequesting(false)}>{CANCEL}</Button>
+            <Button onClick={() => setIsRequesting(false)}>{t("global:cancel")}</Button>
             <Button type="submit" onClick={onSubmit}>
-              {SEND}
+              {t("global:send")}
```

Comment:
With these two, we can probably keep in the global namespace since they are quite generic text, which is a lot more reusable throughout the app than the more specific constants/strings above.